### DISCUSSION TITLE
How to spilt text in two fields

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#15001] [General]
Hi Guys,, i have leads descrioption as name no. (example: description field = john 98767867777 )
i want to split this description via workflow --update fields to so "name" goes to "lead name" and no. goes to mobile field..

is it possible with workflow ?

**Comment by vovpff** [comment]
![image](https://user-images.githubusercontent.com/8545791/126754661-17554907-51d1-496c-bf5f-1661231d885b.png)

**Reply by rioshaz** [reply]
@vovpff thank for taking time buddy,, appreciated but this is for adding two values.
.
but what i want is.. i want to splilt a field value.
 
for ex: 

in my leads form if the description box = john 9876
so now i want move the word "john" to lead name and the number "9876" to mobile no. field...

can this b done...

**Reply by davide-alghi** [reply]
Hi @rioshaz,
split fn is not available. you have to implement it.

**Reply by vovpff** [reply]
Oh. Sorry. My mistake.

I understand what you need. This can't be done with workflows.

You can write custom workflow function what will split for your needs. Or custom handler what will fire after record creation and manage data.

**Reply by rioshaz** [reply]
ooh okay.. any idea how can that be done

**Reply by davide-alghi** [reply]
Hi @rioshaz,

1. create a new workflow task in modules/com_vtiger_workflow/tasks/ folder
2. add a reference to it (a new record) into ```com_vtiger_workflow_tasktypes``` table

**Reply by rioshaz** [reply]
@davide-alghi any step by guide possible buddy...
